.storybook/webpack.config.js

```diff
@@ -7,7 +7,7 @@ const nodeEnvironmentCheck = new webpack.DefinePlugin({
 module.exports = {
   mode: isDev ? 'development' : 'production',
   devtool: isDev ? '#cheap-module-eval-source-map' : 'source-map',
-  plugins: [nodeEnvironmentCheck],
+  plugins: [nodeEnvironmentCheck, new webpack.IgnorePlugin(/^\.\/locale$/, /moment\/min$/)],
```

Comment:
We recently removed this item from the master build because it was killing our storybook stories. Its odd that the preview is working .